lib/Backend/Func.cpp

```diff
@@ -79,6 +79,9 @@ Func::Func(JitArenaAllocator *alloc, CodeGenWorkItem* workItem, const Js::Functi
     hasInlinee(false),
     thisOrParentInlinerHasArguments(false),
     hasStackArgs(false),
+    hasImplicitParamLoad(false),
+    hasThrow(false),
+    hasJitCalls(false),
```

Comment:
>hasJitCalls [](start = 4, length = 11)

if isLeaf is not being used anywhere beyond this phase, do you just want to use and modify the same bool ?